api/files/views.py

```diff
@@ -162,3 +166,83 @@ def get_serializer_context(self):
         context = JSONAPIBaseView.get_serializer_context(self)
         context['file'] = self.file
         return context
+
+
+class FileMetadataRecordsList(JSONAPIBaseView, generics.ListAPIView, FileMixin):
+
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+        PermissionWithGetter(ContributorOrPublic, 'target'),
+    )
+
+    required_read_scopes = [CoreScopes.NODE_FILE_READ]
+    required_write_scopes = [CoreScopes.NODE_FILE_WRITE]
+
+    serializer_class = FileMetadataRecordSerializer
+    view_category = 'files'
```

Comment:
I kept it as "files" since that's the parent route to these routes. It's similar to the versions subroutes which also have a `view_category` of "files".